nomenclature/processor/region.py

```diff
@@ -345,6 +343,7 @@ def apply(self, df: IamDataFrame, dsd: DataStructureDefinition) -> IamDataFrame:
 
                     # Aggregate
                     if self.mappings[model].common_regions is not None:
+                        # processed_dfs_region: List[IamDataFrame] = []
```

Comment:
Is this a mistake?

Reply:
Ah yes, I forgot to remove it. It's taken out now.